src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -494,6 +548,8 @@ function loseFocus() {
     ctaAreaElementFocusable.blur();
   }
 }
+
+const hasLinksInItems = computed(() => props.items.some(item => item.to || item.href));
```

Comment:
This makes dropdown-search depending on logic from ec-btn-dropdown. We don't need tabindexes on `<li>`s if they don't contain links? The native select don't do that either.

Reply:
this means that we cannot focus on any items when opening the dropdown, is that fine?